report.md

```diff
@@ -13,31 +13,41 @@ The focus of the security review was on the following:
 2. Specific security concern 2.
 3. Specific security concern 3.
 
-*Disclaimer*: This security review does not guarantee against a hack. It is a snapshot in time of brink according to the specific commit by a three person team. Any modifications to the code will require a new security review.
+*Disclaimer:* This security review does not guarantee against a hack. It is a snapshot in time of brink according to the specific commit by a three person team. Any modifications to the code will require a new security review.
 
 # Findings 
 
-## High Severity
-### Actual Issue
+## High Risk
+### Issue title (Only first word should be capitalized; titles should never end with punctuation)
 
-**Severity**: Low, Gas optimization
+**Severity:** High
```

Comment:
There is this other PR from Sleepy who wanted to add Critical to the report. Should we do that too?

Reply:
But since it's just an example fine without it as well

Reply:
Added an example section